app/controllers/catalog_controller.rb

```diff
@@ -1860,7 +1860,7 @@ def open_parent_nodes(record)
       parents = [:id => template_to_node_name(record)]
     else
       # Check for parent nodes missing from vandt tree and return them if any
-      parent_rec = ServiceTemplateCatalog.find(record.service_template_catalog_id)
+      parent_rec = ServiceTemplateCatalog.find_by_id(record.service_template_catalog_id) # nil is a valid value
```

Comment:
Ah, miq-bot is right here, this should be `find_by(:id =>`.

It will probably complain again, and not be right that time :).